src/SldStyleParser.ts

```diff
@@ -808,7 +812,7 @@ export class SldStyleParser implements StyleParser<string> {
     }
 
     textSymbolizer.color = color ? color : '#000000';
-    textSymbolizer.opacity = opacity ? numberExpression(opacity) : 1;
+    textSymbolizer.opacity = isNil(opacity) ? 1 : numberExpression(opacity);
```

Comment:
I'm not sure if we should set the opacity at all, if it is not set in the given style, so the default of the used renderer applies. @KaiVolland what do you think?

Reply:
:+1: Forcing an opacity value on the geostyler-style is actually not needed.

Reply:
Ok, so, I'll remove the ternary operator, and only assign the opacity if the parameter is provided

Reply:
Sounds good to me. Please note that this results in a breaking change (which is totally fine). So it would be great, if you could add the information into the commit message.